content/rule_book/scoring.md

```diff
@@ -0,0 +1,299 @@
+# Scoring
+
+## Scoring Individual and Team Points
+
+### Individual Points
+
+All points that occur during a bonus question or during overtime do not contribute to the individual score of a quizzer.
+
+#### Points Earned
+
+- +20 points for every correct question and toss-up question
+- +10 points for every quiz-out without error
+
+#### Points Deducted
+
+- -10 points for 2nd and subsequent personal errors
```

Comment:
Change both uses of "personal" to "individual?"

Reply:
Agreed. 8043240